include/constants/battle_anim.h

```diff
@@ -536,6 +536,7 @@
 #define B_ANIM_BEAK_BLAST_SETUP         33
 #define B_ANIM_SHELL_TRAP_SETUP         34
 #define B_ANIM_ZMOVE_ACTIVATE           35 // Using Z Moves
+#define B_ANIM_AFFECTION_HANGED_ON      36
```

Comment:
```suggestion
#define B_ANIM_AFFECTION_HANGED_ON      36 // TODO: Make new animation, currently a copy of B_ANIM_HANGED_ON.
```

Reply:
I meeean... sure, whatever I guess. That's not what I had in mind at first though 😆